examples/src/operation/bootstrap/bootstrapPool.ts

```diff
@@ -0,0 +1,64 @@
+import { Bootstrap, poolUtils } from "@tinymanorg/tinyman-js-sdk";
+import { getAccount } from "../../util/account";
+import { getAssets } from "../../util/asset";
+import { algodClient } from "../../util/client";
+import signerWithSecretKey from "../../util/initiatorSigner";
+import { SDK_TEST_ARGS } from "../../util/other";
+
+/**
+ * Creates (i.e. "Bootstraps") a pool with an owned asset pair
+ */
+export async function bootstrapPool() {
+  const account = await getAccount();
+  const initiatorAddr = account.addr;
+  const { ids: assetIds } = await getAssets();
+  const [asset1ID, asset2ID] = assetIds;
+  const assetA = await algodClient.getAssetByID(asset1ID).do();
+  const assetB = await algodClient.getAssetByID(asset2ID).do();
+  const [asset_1, asset_2] = [assetA, assetB].map((asset) => ({
+    id: String(asset.index),
+    unit_name: asset.params["unit-name"],
+  }));
+
+  const poolInfo = await poolUtils.v2.getPoolInfo({
+    ...SDK_TEST_ARGS,
+    asset1ID,
+    asset2ID,
+  });
+
+  if (!poolUtils.isPoolNotCreated(poolInfo)) {
+    console.log("⚠️ Pool already exists, skipping bootstrap.");
+    return;
+  }
+
+  const bootstrapTxns = await Bootstrap.v2.generateTxns({
+    ...SDK_TEST_ARGS,
+    asset_1,
+    asset_2,
+    initiatorAddr,
+  });
+
+  const signedTxns = await Bootstrap.v2.signTxns({
+    ...SDK_TEST_ARGS,
+    txGroup: bootstrapTxns,
+    initiatorSigner: signerWithSecretKey(account.sk),
```

Comment:
Within the main README, we have a note under V1 app opt-in with an example of signing with account's secret key. Can you rename the function used here to reinforce that?

```
function signTransactions(txGroups: SignerTransaction[][], accountAddress: string) {
  const toBeSigned = txGroups.flatMap((txGroup) =>
    txGroup.filter((item) => item.signers.includes(accountAddress))
  );
  let signed: Uint8Array[] = [];

  for (let item of toBeSigned) {
    const {blob} = await algosdk.signTransaction(item.txn, accountSecretKey);
    signed.push(blob);
  }

  return signed;
}
```

I'm not sure if this is functional but we can make necessary changes on signTransactions, and also update it on the main README.

Reply:
The one in the README example is not very useful for actually using sdk functions, sdk functions expect:
```ts
initiatorSigner: (txGroupList: SignerTransaction[][]) => Promise<Uint8Array[]>
```
but
```ts
signTransactions: (txGroupList: SignerTransaction[][], accountAddress: string) => Promise<Uint8Array[]>
```


I suggest using this version (I improved mine a bit), and updating README example as well:
```tsx
/**
 * initiatorSigner for signing txns with secret keys
 * @param account account data that will sign the transactions
 */
export default function signerWithSecretKey(account: Account) {
  return function (txGroups: SignerTransaction[][]): Promise<Uint8Array[]> {
    // Filter out transactions that don't need to be signed by the account
    const txnsToBeSigned = txGroups.flatMap((txGroup) =>
      txGroup.filter((item) => item.signers?.includes(account.addr))
    );
    // Sign all transactions that need to be signed by the account
    const signedTxns: Uint8Array[] = txnsToBeSigned.map(({ txn }) =>
      txn.signTxn(account.sk)
    );

    // We wrap this with a Promise since SDK's initiatorSigner expects a Promise
    return new Promise((resolve) => {
      resolve(signedTxns);
    });
  };
}
```

Usage: 
```tsx
initiatorSigner: signerWithSecretKey(account),
```
or
```tsx
const signTransactions = signerWithSecretKey(account);
const signedTxns = signTransactions(txGroup)
```

---

One last thing;
```tsx
 // We wrap this with a Promise since SDK's initiatorSigner expects a Promise
    return new Promise((resolve) => {
// ...
```
I think we can change the type of initiatorSigner in sdk. I believe it's only a Promise if you're using it on client side (awaits user to sign), but if you're using a sk, it's instant.